kube_scheduler/metadata.csv

```diff
@@ -0,0 +1,38 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+kube_scheduler.apiserver.audit.events,count,,,,"Number of audit events generated and sent to the audit backend",0,kube_controller_manager,apiserver.audit.events
+kube_scheduler.apiserver.audit.requests_rejected,count,,,,"Number of apiserver requests rejected due to an error in audit logging backend",0,kube_controller_manager,apiserver.audit.requests_rejected
```

Comment:
Orientation here should be `-1` (lower is better), some other metrics also could benefit for a non-default orientation

Reply:
done added on -1 on others metrics